src/ott/problems/linear/potentials.py

```diff
@@ -157,6 +161,144 @@ def tree_unflatten(
   ) -> "DualPotentials":
     return cls(*children, **aux_data)
 
+  def plot_ot_map(
+      self,
+      source: jnp.ndarray,
+      target: jnp.ndarray,
+      forward: bool = True,
+      alpha: float = 0.5,
+      ax: Optional[matplotlib.axes.Axes] = None,
+      legend_kwargs: Optional[Dict[str, Any]] = None,
+      scatter_kwargs: Optional[Dict[str, Any]] = None,
+  ) -> Tuple[matplotlib.figure.Figure, matplotlib.axes.Axes]:
+    """Plot data and learned optimal transport map.
+
+    Args:
+      source: samples from the source measure
+      target: samples from the target measure
+      forward: use the forward map from the potentials
+        if ``True``, otherwise use the inverse map
+      ax: axis to add the plot to
+      alpha: transparency of the scatter plot points
+      scatter_kwargs: additional kwargs passed into :meth:`~matplotlib.axes.Axes.scatter`
+      legend_kwargs: additional kwargs passed into :meth:`~matplotlib.axes.Axes.legend`
+
+    Returns: matplotlib figure and axis with the plots
```

Comment:
Can you put `matplotlib ...` on new line + indent, like for the `Args:` section? Think this is why it renders weird in the docs.